mrmustard/lab_dev/wires.py

```diff
@@ -111,7 +111,7 @@ def _from_data(cls, id_array=None, modes=None, mask=None):
         w = cls()
         w._id_array = id_array if id_array is not None else w._id_array
         w._modes = modes if modes is not None else w._modes
-        w._mask = mask if mask is not None else w._mask
+        w._mask = mask if mask is not None else np.ones_like(w._id_array)
```

Comment:
👍 did the same

Reply:
I'm thinking of doing a bit more than this when merging wires in dev, because also `w._modes` and `w_id_array` will fail. I think that the solution is changing the signature:
`def _from_data(cls, id_array=None, modes=None, mask=None)` --> `def _from_data(cls, id_array, modes, mask=None)`

What do you think?

Reply:
yeah makes sense